xml/System/String.xml

```diff
@@ -12005,17 +12029,18 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
   
  If `startIndex` is equal to zero, the method returns the original string unchanged.  
    
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```

Comment:
move the substring note to the remarks section of the membergroup element. and review the samples here. There are some missing interactivity.

Reply:
I moved the note.  I looked at all the "Substring#" sample code references and they're all interactive.